Platform.PCL/Realm.Sync.PCL/Permissions/PermissionOfferResponsePCL.cs

```diff
@@ -0,0 +1,86 @@
+﻿////////////////////////////////////////////////////////////////////////////
+//
+// Copyright 2016 Realm Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+////////////////////////////////////////////////////////////////////////////
+
+using System;
+
+namespace Realms.Sync
+{
+    /// <summary>
+    /// Objects of this class are used to accept a <see cref="PermissionOffer"/> using a provided <see cref="Token"/>.
+    /// </summary>
+    /// <remarks>
+    /// Create an instance of <see cref="PermissionOfferResponse"/> using the provided <see cref="PermissionOffer.Token"/> and add it
```

Comment:
Again here. Shorter lines (lines should be readable without scrolling in github :-)